.gitignore

```diff
@@ -1,5 +1,8 @@
 .env
-/build
-/node_modules
-/vendor
+.idea
+*.iml
+/vendor/
+/build/
+node_modules/
 /.php_cs.cache
+js/*hot-update.*
```

Comment:
Since the JS folder will contain compiled JS and the actual src file we're working with are going to be in `src/` wouldn't it make sense to switch this to `js/`, that way we also lose the compiled JS files that are included in this commit. We don't want compiled files in the git repo.